Exercises/NoModules/Chapter 13/Soln13_07/Rational.h

```diff
@@ -0,0 +1,126 @@
+// Rational.h
```

Comment:
I did not verify the math formulas. I'm assuming they are correct...

Reply:
I did the same. Let's have some faith in few-years-ago Peter.